.travis.yml

```diff
@@ -197,6 +197,19 @@ matrix:
     - $TRAVIS_BUILD_DIR/ci/travis_before_script_js.sh
     script:
     - $TRAVIS_BUILD_DIR/ci/travis_script_js.sh
+  # C++ Static w/ gcc 4.9
```

Comment:
I'm really weary of adding yet another CI configuration entry. Parallelism on Travis-CI is ample but limited. I would rather we stop testing all kinds of unusual and weird configurations. We're (or at least some of us are) spending too much time already on build and CI issues, and it's like a sisyphean task.

Reply:
I think static linking is important since there is not even one CI test that does this.
Static linking has been broken many times earlier because of lack of testing. We had a static linking test in the old Parquet repository and it got removed after the mono-repo merge.